spec/helpers/http_mock/nginx_instance.lua

```diff
@@ -14,7 +14,7 @@ local error = error
 local assert = assert
 local ngx = ngx
 local io = io
-local shallow_copy = require "kong.tools.utils".shallow_copy
+local shallow_copy = require("kong.tools.utils").shallow_copy
```

Comment:
If this is changed to `kong.tools.table`, the upgrade test will fail. Seems that there is no `kong/tools/table.lua` file in 2.8.0 codebase.

Reply:
please add a comment for this.

Reply:
Added